src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -391,7 +110,10 @@ def __sdfg__(self, *args, **kwargs) -> dace.sdfg.sdfg.SDFG:
         itir_tmp = itir_transforms.apply_common_transforms(
             self.itir, offset_provider=offset_provider
         )
-        for closure in itir_tmp.closures:  # type: ignore[union-attr]
+        if not isinstance(itir_tmp, itir.FencilDefinition):
+            # FIXME[#1582](kotsaloscv): Update dace orchestration to work with GTIR program
```

Comment:
@kotsaloscv It is not time yet but, as you know, the dace orchestration feature will have to be ported to GTIR. I have left a TODO comment here. This feature is the only piece of code left in the `dace_iterator` module, the new backend is located in `dace_fieldview`.